### ISSUE TITLE
Bug: md-toolbar should have role of toolbar

### ISSUE THREAD

**Issue by bsteffl** [angular/material2#2909]
#### Bug:
Md-toolbar should have `role="toolbar"`

#### What is the expected behavior?
[WCAG Guidelines](https://www.w3.org/TR/wai-aria-practices-1.1/#toolbar)

#### What is the current behavior?
No toolbar role is announced. Would be nice to also provide keyboard functionality for
`Left Arrow` and `Right Arrow`.

#### What are the steps to reproduce?
```
<md-toolbar>
  <span>First Row</span>

  <md-toolbar-row>
    <span>Second Row</span>
  </md-toolbar-row>

  <md-toolbar-row>
    <span>Third Row</span>
  </md-toolbar-row>
</md-toolbar>
```

#### What is the use-case or motivation for changing an existing behavior?
Meet accessibility requirements

#### Which versions of Angular, Material, OS, browsers are affected?
2.0.0-beta.1

**Comment by DevVersion** [comment]
@bsteffl Thanks for the great issue reports! 

Regarding keyboard interaction, I don't think that the toolbar should be responsible for such things. The toolbar is meant to be very simple and should just follow the design guidelines.

**Comment by bsteffl** [comment]
@DevVersion I appreciate your support for accessibility and being inclusive. Keyboard users will still have access to buttons via tabbing so not supporting keyboard actions works too. When I included that I was just thinking about how native OS toolbars work and users understanding of those. Thank you for your contributions.

![image](https://cloud.githubusercontent.com/assets/10760551/22594005/dc3ca2b8-e9e6-11e6-828d-fa109bf00071.png)
